app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -81,6 +106,38 @@ define(function (require, exports, module) {
       return ! account.has('profileImageUrl');
     },
 
+    _addLoadingSpinner: function (spinnerWrapperEl) {
+      return $('<span class="avatar-spinner"></span>').appendTo(spinnerWrapperEl.addClass('with-spinner'));
```

Comment:
@shane-tomlinson do we need guards here for `avatarWrapperEl`, it seems it can be called with `undefined` if no element found in the template, see above.